features/step_definitions/appointment_locator_steps.rb

```diff
@@ -7,12 +8,15 @@
   allow(Locations::Repository).to receive(:all).and_return(locations)
   allow(Geocoder).to receive(:coordinates).with(postcode).and_return(lat_lng)
   allow(Locations::Search).to receive(:nearest_to).with(
-    locations, lat_lng, limit).and_return(locations.take(limit))
+    locations, lat_lng, limit).and_return(search_results.take(limit))
 
   Pages::Locations.new.load(postcode: postcode)
 end
 
 Then(/^I should see the (\d+) appointment locations nearest to that postcode$/) do |number_of_locations|
   expected_locations = number_of_locations.to_i.times.map { |i| "cab-#{i + 1}" }
+  expected_distances = number_of_locations.to_i.times.map { |i| "#{i}km" }
```

Comment:
We should also make sure the number is rounded and we show the correct number of decimal places.